CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 
 ## HEAD
 
+- `gconf` package was reimplemented from scratch
+- `x/cash` is using new `gconf` package for configuration. New genesis path is
```

Comment:
Add

To update genesis file, replace `"gconf": { "cash:xyz": "foo" }` with `"conf": { "cash": { "xyz": "foo" } }`